docs/reference/commandline/build.md

```diff
@@ -457,3 +459,124 @@ space.
 storing two copies of the image, one for the build cache with all the cache
 layers in tact, and one for the squashed version.
 
+#### Prerequisites
+
+The example on this page is using experimental mode in Docker 1.13.
+
+Experimental mode can be enabled by using the `--experimental` flag when starting the Docker daemon or setting `experimental: true` in the `daemon.json` configuration file.
+
+By default, experimental mode is disabled. To see the current configuration, use the `docker version` command.
+
+```none
+
+Server:
+ Version:      1.13.1
+ API version:  1.26 (minimum version 1.12)
+ Go version:   go1.7.5
+ Git commit:   092cba3
+ Built:        Wed Feb  8 06:35:24 2017
+ OS/Arch:      linux/amd64
+ Experimental: false
+
+ [...]
+
+```
+
+To enable experimental mode, users need to restart the docker daemon with the experimental flag enabled.
+
+#### Experimental flag in Ubuntu 14.04:
+
+For Ubuntu 14.04, Docker daemon options are specified as part of upstart system manager. Users can enable experimental mode in Ubuntu 14.04 by changing /etc/default/docker:
+
+```
+/etc/default/docker:
+
+DOCKER_OPTS="--experimental=true"
+
+```
+
+Restart Docker daemon:
+
+```bash
+$ sudo service docker restart
+
+```
+
+Check that Docker is in experimental mode by executing `docker version`:
+
+```
+Server:
+ Version:      1.13.1
+ API version:  1.26 (minimum version 1.12)
+ Go version:   go1.7.5
+ Git commit:   092cba3
+ Built:        Wed Feb  8 06:42:29 2017
+ OS/Arch:      linux/amd64
+ Experimental: true
+
+ [...]
+
+```
+
+#### Experimental flag in Ubuntu 16.04
+
+For Ubuntu 16.04, Docker daemon options are specified as part of systemd system manager. Users can enable experimental mode in Ubuntu 16.04 by editing docker.conf:
```

Comment:
The recommended way to enable experimental features is to use a `daemon.json` configuration file. Doing so works for all init systems (`upstart`, `systemd`), and also when starting the daemon manually. There's a short description of that process here; https://github.com/docker/docker/tree/master/experimental#use-docker-experimental